manheim_cloudmapper/ses/report.py

```diff
@@ -0,0 +1,179 @@
+import os
+import datetime
+import logging
+from premailer import transform
+from .ses import SES
+
+logger = logging.getLogger(__name__)
+
+
+class Report():
+
+    # Base path for CSS stylings, default for cloudmapper
+    BASE_PATH = '/opt/manheim_cloudmapper/web/css'
+
+    def __init__(self,
+                 report_source=(
+                     '/opt/manheim_cloudmapper/web/account-data/report.html'
+                 ),
+                 account_name=None, sender=None, recipient=None,
+                 region=None, ses_enabled=None):
+        """
+        Initialize the Cloudmapper report. Sets variables for email.
+
+        :param account_name: AWS account name
+        :type account_name: str
+        :param sender: Email address registered to SES that will send emails
+        :type sender: str
+        :param recipient: Email address to send emails to
+        :type recipient: str
+        :param region: AWS Region
+        :type region: str
+        :param ses_enabled: wheter the SES email functionality is enabled
+        :type ses_enabled: str
+        """
+        self.report_source = report_source
+        if account_name is None:
+            account_name = os.environ.get('ACCOUNT', None)
```

Comment:
Will this class work properly if ``account_name`` (or sender, recipient, region, etc.) are None? The code here for each of these variables will set it to ``None`` if it's not specified on the class constructor or in the environment, which seems like an error case to me. We should either explicitly raise an exception if any required inputs aren't specified, or alternatively we can just implicitly raise KeyErrors using a pattern like:

```python
if foo is None:
    foo = os.environ['FOO']
```